### ISSUE TITLE
ML: VGG16 Model #1 (24 Layers): Configuration#7: Training model and store the results in the directory for 30 and 50 epochs

### ISSUE THREAD

**Issue by hmehryar** [hmehryar/Hm.JetscapeMl#271]
Model Architect: 

- 4Conv(Conv+Conv+MaxPool+Dropout)
- 4Dense(Dense+Dropout)

1. Put the training on the Grid by using jetscape-ml-tensorflow-cnn.sh
2. save the model and the evaluation data in the power point [slides](https://docs.google.com/presentation/d/1H7D-TKbmZjwNFNcf8Ecuz3HLl1veCZVL/edit?usp=sharing&ouid=108370064464811889896&rtpof=true&sd=true)
3. update the diagrams data on the power point slides

**Comment by hmehryar** [comment]
30 epochs

Epoch 00013: val_accuracy did not improve from 0.86410

840000/840000 [==============================] - 395s 470us/sample - loss: 3.3249 - accuracy: 0.9065 - val_loss: 3.5229 - val_accuracy: 0.6969
Epoch 00013: early stopping

**Comment by hmehryar** [comment]
Epoch 00014: val_accuracy did not improve from 0.88895

840000/840000 [==============================] - 312s 371us/sample - loss: 3.0385 - accuracy: 0.9071 - val_loss: 3.1331 - val_accuracy: 0.7622
Epoch 00014: early stopping

**Comment by hmehryar** [comment]
port into excel for overleaf
